extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -579,28 +579,18 @@ export class SafeframeHostApi {
       if (success || optIsCollapse) {
         this.resizeIframe(height, width);
         this.isCollapsed_ = !!optIsCollapse;
-        this.baseInstance_.element.getResources().resources_.forEach(
-            resource => {
-              if (resource.element == this.baseInstance_.element) {
-                // Need to force a measure event, as measure won't happen immediately
-                // if the element was above the viewport when resize occured, and
-                // without a measure, we'll send the wrong size for the creative
-                // on the geometry update message.
-                resource.measure();
-              }
-            });
+        this.baseInstance_.getResource().measure();
```

Comment:
Are we sure this is async?  Please add a juicy comment on why you're doing this

Reply:
done